tests/e2e/recipe/laravel-boilerplate.php

```diff
@@ -4,8 +4,8 @@
 require_once __DIR__ . '/hosts.php';
 require_once __DIR__ . '/../../../recipe/laravel.php';
 
-set('repository', 'https://github.com/deployphp/test-laravel.git');
-set('branch', 'main');
+set('repository', 'https://github.com/laravel/laravel.git');
+set('branch', 'master');
```

Comment:
When this https://github.com/deployphp/test-laravel/pull/1 PR is merged and created new branch named `laravel-9` apply this suggestion

```suggestion
set('repository', 'https://github.com/deployphp/test-laravel.git');
set('branch', 'laravel-9');
```